en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide-docker-with-apim.md

```diff
@@ -42,27 +33,48 @@ docker ps | grep choreo-connect-
+
+3. Create an API.
+
+     Let's use a mock REST service to create a REST API from scratch.
+ 
+     A mock service with a JSON response `{"hello": "world"}` is provided by default when you use the service URL as `http://run.mocky.io/v2/5185415ba171ea3a00704eed`, which appears in the [https://designer.mocky.io/](https://designer.mocky.io/) mock service. Let's use the HTTP protocol instead of the HTTPS protocol for this guide.
+
+    !!! tip
+        Optionally, to test this service, copy the service URL [http://run.mocky.io/v2/5185415ba171ea3a00704eed](http://run.mocky.io/v2/5185415ba171ea3a00704eed) and navigate to it on a new browser. You should see the following JSON message.
+            
+         `{"hello": "world"}`
+
+4. Select **REST API** from the home screen and then click **Start From Scratch**.
+   
+    [![Design a new REST API]({{base_path}}/assets/img/get_started/design-new-rest-api.png)]({{base_path}}/assets/img/get_started/design-new-rest-api.png)
+
+5. Enter the API details.
+     
+     [![Create an API]({{base_path}}/assets/img/get_started/api-create.png){: style="width:60%"}]({{base_path}}/assets/img/get_started/api-create.png)
 
-1. First sign in to the API Manager Publisher Portal by accessing the URL: https://apim:9443/publisher/
+6. Click **Create & Publish**.
 
-2. Click on the REST API card and then click on the `Deploy Sample API` button. This will deploy the sample PizzaShack API.
-    [![]({{base_path}}/assets/img/deploy/mgw/deploy-sample-api.png)]({{base_path}}/assets/img/deploy/mgw/deploy-sample-api.png)
+     This will publish your first API on the Developer Portal as well as deploy it on Choreo Connect. You now have an OAuth 2.0 secured REST API that is ready to be consumed.
 
-3. Click on `Endpoints` from the left menu inside the PizzaShackAPI. Change the production endpoint and
-sandbox endpoint URL to `http://apim:9763/am/sample/pizzashack/v1/api/`. Save the changes by clicking the **Save** button.
-    [![]({{base_path}}/assets/img/deploy/mgw/endpoint-tab.png)]({{base_path}}/assets/img/deploy/mgw/endpoint-tab.png)
-    [![]({{base_path}}/assets/img/deploy/mgw/endpoint-edit.png)]({{base_path}}/assets/img/deploy/mgw/endpoint-edit.png)
+### Step 4 - Subscribing to API and Get a Token
 
-4. Follow the [documentation on deploying an API]({{base_path}}/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-an-api) to deploy 
-the changes done to the API into Choreo Connect.
+1. Navigate to the Developer Portal and and click an API (e.g., `HelloWorld`).
 
-### Step 5 - Subscribing to an API and getting a token
+    [https://apim:9443/devportal/](https://apim:9443/devportal/)
 
-1. Sign in to the WSO2 Developer Portal (`https://<hostname>:9443/devportal`) and click an API (e.g., `PizzaShack`).
+2. Subscribe to the API (e.g., `HelloWorld` 1.0.0) using an application and an available throttling policy.
```

Comment:
"2. Subscribe to the API (e.g., `HelloWorld` 1.0.0) using an application and an available Rate Limiting Policy."